deployments/examples/ocis_individual_services/docker-compose.yml

```diff
@@ -356,12 +519,17 @@ services:
       GRAPH_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
 
       GRAPH_HTTP_ADDR: 0.0.0.0:9120
-      GRAPH_SPACES_WEBDAV_BASE: https://${OCIS_DOMAIN:-ocis.owncloud.test}
+      GRAPH_SPACES_WEBDAV_BASE: https://${OCIS_DOMAIN}
+
+      GRAPH_LDAP_URI: ldaps://idm:9235
+      GRAPH_LDAP_BIND_PASSWORD: ${IDM_SVC_PASSWORD}
+      GRAPH_LDAP_INSECURE: true # TODO: fix me https://github.com/owncloud/ocis/issues/3818
```

Comment:
The "old" python-based `docker-compose` doesn't like this. Works with `GRAPH_LDAP_INSECURE: "true"`
Same is true for the `PROXY_INSECURE_BACKEND`. Not sure if if still want to support that, though.

Reply:
I think we should not support the "old" `docker-compose`.